src/rust/ide/ast/impl/src/known.rs

```diff
@@ -98,6 +98,14 @@ impl<T:Into<Shape<Ast>>> KnownAst<T> {
     }
 }
 
+impl KnownAst<crate::Module<Ast>> {
+    /// Creates a new `KnownAst<Module>` from `shape`.
```

Comment:
True but this doesn't really explain the purpose of this function, i.e. that modules require some special treatment.
(if they do)